### ISSUE TITLE
train the model from scratch

### ISSUE THREAD

**Issue by qiulesun** [vlfeat/matconvnet#908]
I want to train the vgg-vd-16 model from scratch, which parameter initialization method should I choose, gaussian or xavier or xavierimproved?

**Comment by ankush-me** [comment]
If you use batch-normalization, then training is not too sensitive to the initialization you use; i.e., with batch-normalization, any / all of the above methods should work.

**Comment by qiulesun** [comment]
Thank you for your answer.I use the batch-normalization,but the top1err decreased only from 0.99 to 0.91 after 5 epochs, and the convergence rate was slow.Is this normal?
